__tests__/plugin.test.ts

```diff
@@ -72,7 +72,21 @@ describe('babel plugin', () => {
       </script>`;
 
       const { code } = runPlugin(input, {});
-      expect(code).toContain(`f.__version = "${packageVersion}";`);
+      expect(code).toContain(`version: "${packageVersion}"`);
+    });
+
+    it('injects source maps', () => {
+      delete process.env.REANIMATED_JEST_MOCK_SOURCEMAP;
+
+      const input = html`<script>
+        function foo() {
+          'worklet';
+          var foo = 'bar';
+        }
+      </script>`;
+
+      const { code } = runPlugin(input, { sourceMaps: true });
+      expect(code).toContain('{\\"version\\":3');
```

Comment:
Why do we check here if the `version` property exists instead of checking that the `sourceMap` property exists?

Reply:
Changed in 47e279f8d13fb32ac4c17cc99d999c19ab4b4b7f